components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -526,6 +517,13 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
   }
 }
 
+- (void)markIntrinsicContentSizeAndLayoutNeedingUpdateForSelfAndItemView:(UIView *)itemView {
```

Comment:
Nit: Can this be `invalidateLayoutForSelfAndItemView:`

Reply:
It could, but that would only be `setNeedsLayout`, yeah? I know naming is hard but in this case I feel it's important that the method capture that `intrinsicContentSize` is no longer valid.  That said, if you feel strongly I can certainly change it.